### DISCUSSION TITLE
Hang of systemd-logind.service on poweroff?

### DISCUSSION THREAD

**Discussion by dev-zero** [kanidm/kanidm#2313] [Q&A]
We have just noticed, that `systemd-logind.service` occasionally hangs on nodes where we rely on `kanidm-unixd.service`, although we could not fully determine yet whether this is really correlated.

What I have noticed is that services like `sssd` for example have this:

```
Before=systemd-user-sessions.service nss-user-lookup.target
Wants=nss-user-lookup.target
```

and now I wonder whether `kanidm-unixd` should have the same?

**Comment by yaleman** [comment]
If it's hanging on power *off* then I'm not sure how the service dependencies would come into it?

**Reply by Firstyear** [reply]
Because deps also provide shutdown order, and so here systemd could be relying on a user session to exist and kanidm unix is already shutdown

I see no issue to add these to the unit files.

**Comment by Firstyear** [comment]
Resolved in #2314
